device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnectionTest.java

```diff
@@ -2226,17 +2226,17 @@ public void OnLinkRemoteCloseReportsErrorCodeIfPresent() throws TransportExcepti
                 mockEvent.getReceiver();
                 result = null;
                 mockEvent.getTransport();
-                result = mockTransport;
+                result = null;
```

Comment:
This test is for checking the link for an error condition. If the transport is not null, it will try to check the transport for an error condition instead.